src/pages/pt-BR/comparing-astro-vs-other-tools.md

```diff
@@ -0,0 +1,298 @@
+
+## Gatsby vs. Astro
+
+[Gatsby](https://www.gatsbyjs.com/) é um popular framework de websites e aplicações para React.
+
+Gatsby utiliza React para renderizar seu website. Astro é mais flexível: você está livre para construir sua UI com qualquer biblioteca de componentes popular (React, Preact, Vue, Svelte, Solid e outros) ou a sintaxe de componentes do Astro que é similar a HTML + JSX.
+
+Gatsby v4 suporta tanto a Geração Estática de Sites (SSG) com rebuilds incrementais, a Geração Estática Deferida (DSG) e Renderização no Lado do Servidor (SSR). Astro pode fazer builds estaticamente via SSG, ou passar para deploy em ambientes de SSR via [adaptadores](/pt-BR/guides/server-side-rendering/#habilitando-o-ssr-em-seu-projeto): Deno, serverless da Vercel, serverless da Netlify e Node.js, com mais vindo no futuro.
+
+Gatsby requer uma API customizada de GraphQL para funcionar com todo o conteúdo do seu site. Enquanto alguns desenvolvedores gostam desse modelo, uma crítica comum ao Gatsby é que esse modelo se torna muito complexo e difícil de se manter conforme o tempo, especialmente em relação ao quanto os sites crescem. Astro não necessita de GraphQL, providenciando APIs familiares (como `fetch()` ou top-level `await`) para carregamento de dados próximo de onde os dados são necessários. Porém, você está livre para escolher utilizar qualquer biblioteca GraphQL no lado do servidor ou no lado do cliente com Astro.
+
+#### Comparando a Performance do Gatsby vs. Astro
+
+Na maioria dos casos, websites Astro irão carregar significantemente mais rápido que websites Gatsby. Isso acontece pois Astro automaticamente remove qualquer JavaScript desnecessário da página, hidratando apenas os componentes que o precisam individualmente. Essa funcionalidade é chamada de [hidratação parcial](/pt-BR/core-concepts/partial-hydration/).
+
+Gatsby não suporta hidratação parcial, e ao invés disso faz o usuário carregar e hidratar novamente a página inteira no navegador, mesmo que a maioria do conteúdo da página seja estático. Isso cria um carregamento mais lento da página e piora a performance do seu website. Gatsby tem um [plugin da comunidade](https://www.gatsbyjs.com/plugins/gatsby-plugin-no-javascript/) para remover todo o JavaScript da página, porém isso quebraria muitos websites. Isso te deixa numa situação de tudo ou nada com a interatividade de cada página.
+
+Gatsby tem um ótimo ecossistema de plugins, enquanto a [coleção de integrações](https://astro.build/integrations/) do Astro é menor, porém em constante crescimento. Gatsby provém o plugin [gatsby-plugin-image](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/) para otimizações avançadas de imagens. Enquanto Astro não tem uma solução própria comparável, [astro-imagetools](https://github.com/RafidMuhymin/astro-imagetools#readme) é uma integração popular da comunidade para otimização de imagens, imagens de fundo e geração de imagens responsivas.
```

Comment:
I agree with the changing `provém` to `fornece`, but changing `Enquanto` to `Apesar` deviates from the original text since we are now only saying "We don't have an official solution" instead of "We don't have an official solution, yet"

Reply:
fair enough